### PR TITLE
[Shop][Checkout] Preselect country in addressing form

### PR DESCRIPTION
**Pull request by Wojdylak**
| Q               | A
|-----------------|-----
| Branch?         | bootstrap-shop
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | n/a
| License         | MIT

### CODE REVIEW

File: src/Sylius/Bundle/ShopBundle/templates/shared/form/address/province.html.twig [hunk 1]
```diff
@@ -1,13 +1,10 @@
 {% set form = hookable_metadata.context.form %}
 
-<div data-url="{{ path('sylius_shop_ajax_render_province_form') }}">
+<div>
     {% if form.provinceCode is defined %}
         {{ form_row(form.provinceCode, sylius_test_form_attribute('province-code')|sylius_merge_recursive( {'attr': {'class': ''}})) }}
+        {{ form_errors(form.provinceCode) }}
```
Comment (NoResponseMate) [inline src/Sylius/Bundle/ShopBundle/templates/shared/form/address/province.html.twig]: Isn't this included in the `form_row` above?

Comment (Wojdylak) [reply src/Sylius/Bundle/ShopBundle/templates/shared/form/address/province.html.twig]: You are right

**Review by TheMilek — APPROVED** [review]

File: src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php [hunk 2]
```diff
@@ -1064,6 +1064,22 @@ public function iShouldBeNotifiedThatTheAndTheInShippingDetailsAreRequired(strin
...
+     */
+    public function iShouldBeNotifiedThatTheInShippingDetailsIsRequired(string $element, string $type): void
+    {
+        /** @var array|null $violations */
+        $violations = $this->responseChecker->getResponseContent($this->client->getLastResponse())['violations'];
+        $type .= 'Address';
+
+        $violation = $this->getViolation(
+            $violations,
+            $type . '.' . StringInflector::nameToCamelCase($element),
```
Comment (GSadee) [inline src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php:1078]:
```suggestion
        $violation = $this->getViolation(
            $violations,
            $type . 'Address.' . StringInflector::nameToCamelCase($element),
```

File: src/Sylius/Behat/Context/Ui/Shop/Checkout/CheckoutAddressingContext.php [hunk 3]
```diff
@@ -152,6 +151,14 @@ public function iSpecifyTheShippingAddressAs(AddressInterface $address): void
         $this->addressPage->specifyShippingAddress($address);
     }
 
+    /**
+     * @When /^I (do not specify any shipping address) information$/
```
Comment (GSadee) [inline src/Sylius/Behat/Context/Ui/Shop/Checkout/CheckoutAddressingContext.php:155]:
```suggestion
     * @When I do not specify any shipping address information
```

**Review by GSadee — APPROVED** [review]

**Pull request merged**